### PR TITLE
added checklist to success

### PR DESCRIPTION
**Pull request by CrystalOnScript**

### CODE REVIEW

File: pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+If you're interested in testing how introducing AMP to your website impacts your users, make sure you're setting up testing to measure the correct things. Its likely to see false negative, positive or relevant results if your analytics don't account for the differences AMP creates in measurement. 
+
+Learn more about setting up proper analytics for AMP:
+
+
+
+*   [So your AMP test doesn't perform — now what?](https://blog.amp.dev/2018/11/08/so-your-amp-test-doesnt-perform%e2%80%8a-%e2%80%8anow-what/)
+*   [Cache vs. non-cache analysis](https://support.google.com/analytics/answer/6343176?hl=en#cache)
+*   [Measuring user journeys across the AMP Cache and your website](https://www.ampproject.org/latest/blog/measuring-user-journeys-across-the-amp-cache-and-your-website/)
```
Comment (sebastianbenz) [inline pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md]: Link to amp.dev

File: pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+*   [The AMP Validator ](https://validator.ampproject.org/)
+*   [AMP Tools]({{g.doc('/content/amp-dev/documentation/documentation/tools.html', locale=doc.locale).url.path}}?format=websites) 
+
+
+# Grant cached AMP pages server access
+
+Valid AMP pages are automatically opted into all existing AMP caches. This is great news! Your pages are automatically preloaded efficiently and safely, enabling user-beneficial performance optimizations to content. However, documents served from AMP caches are served from domains that do not match your own. This can cause  Cross-Origin Resource Sharing (CORS) issues when using dynamic AMP components, such as [`<amp-form>`]({{g.doc('/content/amp-dev/documentation/components/reference/amp-form.html', locale=doc.locale).url.path}}?format=websites) or [`<amp-list>`]({{g.doc('/content/amp-dev/components/reference/amp-list.html', locale=doc.locale).url.path}}?format=websites), causing your site to lose access to your data. This means certain content may not load correctly or at all.
```
Comment (sebastianbenz) [inline pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md]: * Mention that it's important to enable CORS Requests from all available AMP Caches: https://cdn.ampproject.org/caches.json
* Maybe link to the [AMP CORS lib](https://www.npmjs.com/package/amp-toolbox-cors) which implements all the best practices

File: pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+*   [How AMP Pages are cached ]({{g.doc('/content/amp-dev/documentation/guides-and-tutorials/learn/amp-caches-and-cors/how-cached.html', locale=doc.locale).url.path}}?format=websites)
+*   [CORS in AMP]({{g.doc('/content/amp-dev/documentation/guides-and-tutorials/learn/amp-caches-and-cors/amp-cors-requests.html', locale=doc.locale).url.path}}?format=websites)
+
+
+# Set up discoverable canonicals 
+
+Whether you're only using AMP or taking the paired-AMP approach, ensure that your AMP pages are discoverable by users. Paired AMP pages with non-AMP pages will need to link to each other in their `<head>`. If using a single AMP page, it will need to link to itself. Additionally, [Schema.org](https://schema.org/) metadata adds useful information other sites or search engines may require to share your content. 
+
+Web robots (also known as Web Wanderers, Crawlers, or Spiders) are programs that traverse the Web automatically. They have many uses, such as helping search engines index the web content. Ensure your sites `robots.txt` file has the proper instructions for them to follow, or set up the appropriate headers.
```
Comment (sebastianbenz) [inline pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md]: Maybe add a few examples from here: https://medium.com/google-developers/how-to-avoid-common-mistakes-when-publishing-accelerated-mobile-pages-9ea61abf530f (item 3.)

**Comment by sebastianbenz** [comment]
@matthiasrohmer https://ampbyexample.com/introduction/how_to_publish_amps/ should redirect to this article

File: pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md (hunk 3 above)
Comment (alabiaga) [reply pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md]: +1

File: pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md (hunk 2 above)
Comment (alabiaga) [reply pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md]: > Mention that it's important to enable CORS Requests from all available AMP Caches: cdn.ampproject.org/caches.json

Yea this is good to mention though I am not sure about the snippet about enabling it for all. Though yes that would be ideal and most correct. I would just be careful about the wording as enabling it for all is not a prerequisite for getting it to work. Maybe note that they should be mindful of the all the available AMP caches and to configure their servers to allow for the ones for their particular needs.

**Comment by CrystalOnScript** [comment]
@sebastianbenz I updated with documentation on testing the Google AMP Cache (and added you as an author). Do you think its overkill? Thanks!

File: pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md [hunk 4]
```diff
@@ -20,27 +22,80 @@ Learn more about the tools you can use to validate your AMP pages:
...
 Learn more about accessing server data:
 
 
 
 *   [How AMP Pages are cached ]({{g.doc('/content/amp-dev/documentation/guides-and-tutorials/learn/amp-caches-and-cors/how-cached.html', locale=doc.locale).url.path}}?format=websites)
 *   [CORS in AMP]({{g.doc('/content/amp-dev/documentation/guides-and-tutorials/learn/amp-caches-and-cors/amp-cors-requests.html', locale=doc.locale).url.path}}?format=websites)
+*   [AMP CORS Library](https://www.npmjs.com/package/amp-toolbox-cors)
 
 
+#Test pages in the Google AMP Cache
```
Comment (sebastianbenz) [inline pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md]: nit: space after `#`

File: pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+
+*   [How AMP Pages are cached ]({{g.doc('/content/amp-dev/documentation/guides-and-tutorials/learn/amp-caches-and-cors/how-cached.html', locale=doc.locale).url.path}}?format=websites)
+*   [CORS in AMP]({{g.doc('/content/amp-dev/documentation/guides-and-tutorials/learn/amp-caches-and-cors/amp-cors-requests.html', locale=doc.locale).url.path}}?format=websites)
+*   [AMP CORS Library](https://www.npmjs.com/package/amp-toolbox-cors)
+
+
+#Test pages in the Google AMP Cache
+The Google AMP Cache stores valid AMP pages and provides consistently fast access to them. The [Google Top Stories Carousel with AMP](https://developers.google.com/search/docs/guides/mark-up-content#use-amp-html), for example, uses the Google AMP Cache to display articles. The cache stores images and fonts in addition to documents. This makes it important to test that your AMPs work correctly when loaded via the Google AMP Cache.
+
+{{ image('/static/img/docs/amp_error.png', 385, 386, layout='intrinsic', alt='Image showing an error from the Google AMP Cache') }}
```
Comment (sebastianbenz) [inline pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md]: I wouldn't add this image as it shouldn't happen anymore.

File: pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+
+Loading your AMP pages via the Google AMP Cache is easy. The Google AMP Cache URL is composed based on whether the source URL is available via HTTP or HTTPS:
```
Comment (sebastianbenz) [inline pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md]: I think the whole section is a bit redundant to 

https://github.com/ampproject/docs/blob/future/pages/content/amp-dev/documentation/guides-and-tutorials/learn/amp-caches-and-cors/amp-cache-debugging.md

and

https://github.com/ampproject/docs/blob/future/pages/content/amp-dev/documentation/guides-and-tutorials/learn/amp-caches-and-cors/amp-cache-urls.md

I'd instead link to these/

File: pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+When loading your AMP pages via the Google AMP Cache, check via your [browser’s developer tools](https://developers.google.com/web/tools/chrome-devtools/) if all external resources can be loaded successfully, including all of the following:
+
+* images
+* videos
+* amp-analytics endpoints
+* amp-pixel endpoints
+* custom fonts
+* iframes
+
+A common cause for missing assets are [protocol relative URLs](https://en.wikipedia.org/wiki/Wikipedia:Protocol-relative_URL). These are currently not supported by the Google AMP Cache. Instead link to all assets via HTTPS (if available).
```
Comment (sebastianbenz) [inline pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md]: TBH, I'm not sure that this is still a problem. I'd suggest removing.

File: pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+
+
+
+*   [How AMP Pages are cached ]({{g.doc('/content/amp-dev/documentation/guides-and-tutorials/learn/amp-caches-and-cors/how-cached.html', locale=doc.locale).url.path}}?format=websites)
+*   [CORS in AMP]({{g.doc('/content/amp-dev/documentation/guides-and-tutorials/learn/amp-caches-and-cors/amp-cors-requests.html', locale=doc.locale).url.path}}?format=websites)
+*   [AMP CORS Library](https://www.npmjs.com/package/amp-toolbox-cors)
+
+
+#Test pages in the Google AMP Cache
+The Google AMP Cache stores valid AMP pages and provides consistently fast access to them. The [Google Top Stories Carousel with AMP](https://developers.google.com/search/docs/guides/mark-up-content#use-amp-html), for example, uses the Google AMP Cache to display articles. The cache stores images and fonts in addition to documents. This makes it important to test that your AMPs work correctly when loaded via the Google AMP Cache.
```
Comment (sebastianbenz) [inline pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md]: I'd remove the Google specific part. Maybe only:

```
AMP Cache stores valid AMP pages and provides consistently fast access to them. Cache stores images and fonts in addition to documents. This makes it important to test that your AMPs work correctly when loaded via an AMP Cache.
```

**Comment by CrystalOnScript** [comment]
Updates again @sebastianbenz PTAL :)

**Pull request merged**